16/feoh/randomtitlepicker.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import random
+# You'll need to create a file named config.py containing a line like: apikey="<your_api_key>"
+from config import apikey
+
+# Random OMDB title picker.
+
+
+def generate_random_imdb_id():
+    # This is the highest IMDB title id at time of writing. This stinks. Wish I had a better way to
+    # dynamically set this!
+    max_imdb_title_number = 9916754
+    random.seed()
+    random_id = int(random.random() * max_imdb_title_number)
+    return f"tt{random_id}"
+
+
+def make_omdb_request(imdb_title_id):
+    movie_params = {'apikey': apikey, 'i': imdb_title_id}
+    return requests.get("http://www.omdbapi.com", movie_params)
+
+
+def get_random_movie() -> dict:
+    usable_movie = False
+    movie_json = {}
+
+    while True:
+        random_imdb_title_id = generate_random_imdb_id()
+        req = make_omdb_request(random_imdb_title_id)
+        movie_json: dict = req.json()
```

Comment:
so nice requests support the `json()` method no?

Reply:
huh what is that `: dict` syntax?

Reply:
It's a type hint. Pycharm was trying to get me to use them. I've since decided I disagree with it :) IMO it makes the code less readable (to humans :) and doesn't gain you much.